README.md

```diff
@@ -255,7 +256,10 @@ will receive a `[]interface{}` slice, with values of the following types:
 * `json.Number` for any numeric Trino types
 * `[]interface{}` for Trino arrays
 * `map[string]interface{}` for Trino maps
-* `string` for other Trino types, as character, date, time, or timestamp
+* `string` for other Trino types, as character, date, time, varbinary or timestamp
```

Comment:
This section is about reading data from Trino. We discussed this in another comment that support for []byte will be added in a separate PR, so remove this for now

Reply:
oh yes, it removed now